db/schema.rb

```diff
@@ -50,8 +50,8 @@
     t.string   "kind"
     t.boolean  "optional",     :default=>false
     t.string   "requirements"
-    t.datetime "created_at",   :null=>false
-    t.datetime "updated_at",   :null=>false
+    t.datetime "created_at"
```

Comment:
Why are these definitions shifting?

Reply:
I'm not sure... I just re-did the migration from a fresh master schema.db and it generated the same file...

Reply:
That's my usual procedure as well, however I think I'm running into this https://github.com/SchemaPlus/schema_plus_core/issues/22 locally, and I cannot reset the db.